pci/src/vfio.rs

```diff
@@ -390,13 +390,19 @@ struct VfioCommonState {
 
 impl VersionMapped for VfioCommonState {}
 
+pub(crate) struct ConfigPatch {
+    mask: u32,
```

Comment:
Does this create the opportunity to patch any of the PCI Express configuration space from command line? I think that would be pretty powerful, that is, to offer the ability to block specific offsets from being written...

Reply:
Well yes you could definitely provide this ability to the user, but I don't think we want to allow such thing though.

Reply:
@sboeuf Wishing to follow up on this comment. I do have a desire, but not sure a need, to block write access to specific PCIE MMIO. You mentioned you were not convinced we want to allow such a behavior.

Can you elaborate on why this would be undesireable? I realize the concept of forcing certain parts of MMIO to read-only may be difficult to work with or use and negatively impact usability.

If the functionality is used to ensure functional safety, is it worth the tradeoff in usability?